fluent-package/Rakefile

```diff
@@ -596,16 +585,16 @@ class BuildTask
       end
 
       desc "Create configuration files for Red Hat like systems with systemd"
-      task :rpm_config => [:fluent_package_config, :systemd_tmpfiles_config, :bin_scripts, :rpm_systemd]
+      task :rpm_config => [:rpm_fluent_package_config, :systemd_tmpfiles_config, :bin_scripts, :rpm_systemd]
 
       desc "Create configuration files for Debian like systems"
-      task :deb_config => [:fluent_package_config, :systemd_tmpfiles_config, :bin_scripts, :deb_systemd, :deb_scripts]
+      task :deb_config => [:deb_fluent_package_config, :systemd_tmpfiles_config, :bin_scripts, :deb_systemd, :deb_scripts]
 
       desc "Create configuration files for Windows"
-      task :msi_config => [:fluent_package_config, :wix_config, :win_batch_files]
+      task :msi_config => [:rpm_fluent_package_config, :wix_config, :win_batch_files]
 
       desc "Create configuration files for macOS"
-      task :dmg_config => [:fluent_package_config, :pkgbuild_scripts, :pkgbuild_config, :bin_scripts, :launchctl_config]
+      task :dmg_config => [:rpm_fluent_package_config, :pkgbuild_scripts, :pkgbuild_config, :bin_scripts, :launchctl_config]
```

Comment:
For future tasks, we need to improve these codes by separating the role of the `render_fluent_package_config` method.
First of all, I think there should be variables for the user and group.
(I think it should be done in another PR.)